doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -250,6 +250,9 @@ Bluetooth samples
 
   * The :ref:`channel_sounding_ras_reflector` sample demonstrating how to implement a Channel Sounding Reflector that exposes the Ranging Responder GATT Service.
   * The :ref:`channel_sounding_ras_initiator` sample demonstrating basic distance estimation with Channel Sounding by setting up a Channel Sounding Initiator that acts as a Ranging Requestor GATT Client.
+  * Support for the :ref:`zephyr:nrf54l15dk_nrf54l05` and :ref:`zephyr:nrf54l15dk_nrf54l10` board targets in the following samples:
+
+      * :ref:`peripheral_hids_mouse`
```

Comment:
```suggestion
  * Support for the :ref:`zephyr:nrf54l15dk_nrf54l05` and :ref:`zephyr:nrf54l15dk_nrf54l10` board targets in the following sample:

    * :ref:`peripheral_hids_mouse`
```
We could place this here with the understanding that more Bluetooth samples will have the support.

Reply:
fixed